python/mxnet/contrib/onnx/onnx2mx/_op_translations.py

```diff
@@ -118,11 +118,23 @@ def add_n(attrs, inputs, proto_obj):
 # Sorting and Searching
 def argmax(attrs, inputs, proto_obj):
     """Returns indices of the maximum values along an axis"""
-    return 'argmax', attrs, inputs
+    new_attrs = translation_utils._fix_attribute_names(attrs, {'axis': 'axis',
```

Comment:
what is this statement doing? it seems redundant.

Reply:
removed